jupyterhub/tasks/main.yml

```diff
@@ -274,7 +263,7 @@
       c.BatchSpawnerBase.req_runtime = '{{ req_runtime }}'
       c.BatchSpawnerBase.req_memory = '{{ req_memory }}'
       c.BatchSpawnerBase.env_keep = ['LANG','JUPYTERHUB_API_TOKEN','JPY_API_TOKEN']
-      c.SlurmSpawner.batch_script = """#!/bin/bash
+      c.SlurmSpawner.batch_script = ""#!/bin/bash
```

Comment:
Why was the single quote removed here? Triple-quotes are a way to span strings across multiple lines in python.

Reply:
I was paying too much attention to the format color of the file in my editor. When I removed the extra quote, the formatting of the color showed that the issue was corrected.  Note to self not to rely on an editor's color formatting in pointing out issues.